distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1087,23 +1085,23 @@ public void persistObjectToWorkspace() throws Exception {
     @Test
     public void testValidationEnforced() throws Exception {
         getServiceManager().startFeature(true, "catalog-plugin-metacard-validation");
+        // Update metacardMarkerPlugin config with enforcedMetacardValidators
```

Comment:
It looks like you update this configuration in multiple tests.  I would suggest extracting a helper method with the common code.